modtools/create-unit.lua

```diff
@@ -566,6 +587,7 @@ if args.name then
 else
   local unit = df.unit.find(unitId)
   unit.name.has_name = false
+  unit.name.first_name = "" -- remove the first name altogether. gets rid of units having number names.
```

Comment:
Awesome I'll look at updating rd hack after the release this week in the MW repository and run tests on it this week... I've declared a break in that release schedule anyways.